modules/oe_content_skos_person_reference/oe_content_skos_person_reference.info.yml

```diff
@@ -0,0 +1,18 @@
+name: OpenEuropa Content SKOS Person Reference
+description: Contains SKOS Person Reference field storage and the Person sub-entity bundle Political leader.
+package: OpenEuropa
+type: module
+core_version_requirement: ^9.4 || ^10
+
+dependencies:
+  - oe_content:oe_content_sub_entity_person
+
+config_devel:
+  install:
+    - core.entity_form_display.oe_person.oe_political_leader.default
+    - core.entity_view_display.oe_person.oe_political_leader.default
+    - field.field.oe_person.oe_political_leader.oe_skos_reference
+    - field.storage.oe_person.oe_skos_reference
+    - oe_content_sub_entity_person.oe_person_type.oe_political_leader
+  optional:
+    - language.content_settings.oe_person.oe_political_leader
```

Comment:
These configs were moved from the module, they need to be revisited and the dependencies inside these configs need to be updated to the new ones.

Reply:
I removed the config_devel list of configs that were moved to tasks/install in [4813864](https://github.com/openeuropa/oe_content/pull/591/commits/48138644c88a2c756c9ef486586e7efb38f41d58).  I double checked the configs and dependencies and it looked good to me (I also did a rebase and a re-export of these configs but didn't see any difference).